examples/01_02_why_async/Cargo.toml

```diff
@@ -7,4 +7,4 @@ edition = "2018"
 [lib]
 
 [dev-dependencies]
-futures-preview = { version = "=0.3.0-alpha.17", features = ["async-await", "nightly"] }
+futures = "=0.3.1"
```

Comment:
These no longer need the leading `=`

Reply:
Good point, addresssed - ready for review. Thank you